packages/main/src/plugin/api/provider-info.ts

```diff
@@ -115,3 +119,10 @@ export interface PreflightCheckEvent {
   type: 'start' | 'stop';
   status: CheckStatus;
 }
+
+export interface ProviderCleanupActionInfo {
```

Comment:
I don't see where it's been used

Reply:
I was assuming UI will require to display details about each action but as for now nothing is shown yes it's not used
I can remove it and add it later

Reply:
removed

Reply:
so it looks like it's used in provider-registry.ts

put it back

Reply:
it is used there: https://github.com/containers/podman-desktop/blob/f49d5c7d15e805aa8bf8a79562936aa5a66ef446/packages/main/src/plugin/provider-registry.ts#L493-L521

Reply:
ping @jeffmaury